### PR TITLE
Proba v update

### PR DESCRIPTION
**Pull request by mitjak**
https://git.sinergise.com/team-6/eobrowser3/-/issues/1040

### CODE REVIEW

File: src/layer/wms.ts [hunk 1]
```diff
@@ -89,16 +89,16 @@ export function wmsGetMapUrl(
   }
 
   if (!params.fromTime) {
-    queryParams.time = moment.utc(params.toTime).format('YYYY-MM-DDTHH:mm:ss') + 'Z';
+    queryParams.time = moment.utc(params.toTime).format('YYYY-MM-DD');
```
Comment (dgostencnik) [inline src/layer/wms.ts:92]: why was this changed to date only?

Comment (mitjak) [reply src/layer/wms.ts:92]: Proba-V doesn't support time in single date mode. Setting this universally makes sense as we're actually searching for date and not datetime in this case. Other providers support this as well.

Comment (dgostencnik) [reply src/layer/wms.ts:103]: I think this should be handled by client (eo-browser in this case) and not by library. WMS handler should be agnostic and  doesn't know anything about proba-v or any other wms service. Just because eo-browser uses some wms service which doesn't support time,  that doesn't mean that 
- all wms services supports date only
- no one will ever want to set time, after all parameter is called `time` and not date

Personally, I would do this in eob

Comment (mitjak) [reply src/layer/wms.ts]: I generally agree. But am also not sure EOB is the right place to handle these peculiarities. All in all the right way of addressing these would be to read getCapabilities more rigorously and then act accordingly (not sure if it does indeed contain all the required info). But am not sure if this is something worth pursuing at this point. 

Perhaps I would rather suggest creating a separate layer for the WMT MS and handle it there. What do you think?

Comment (dgostencnik) [reply src/layer/wms.ts]: Yes, I think this could work.

File: src/layer/LayersFactory.ts [hunk 2]
```diff
@@ -332,6 +340,36 @@ export class LayersFactory {
     );
   }
 
+  private static async makeLayersWmsWmtMs(
```
Comment (dgostencnik) [inline src/layer/LayersFactory.ts:343]: This could be merged with `makeLayersWms`as  it is duplicated except for the layer class.  I think one function with additional param layerClass would be better. 

Or perhaps even better option would be to  change  `fetchLayersFromGetCapabilitiesXml` a bit (extract parsing capabilities into own function) and use `fetchGetCapabilitiesXml` to get capabilities and determine with isWMSCapabilities which layer should be created. In this scenario we could also remove hardcoded url  above (services.terrascope.be/wms/v2)

**Review by dgostencnik — APPROVED** [review]
